sktime/param_est/cointegration/_johansen.py

```diff
@@ -0,0 +1,218 @@
+# needs more parameters from the original
+"""Parameter estimation for cointegration."""
+
+author = ["PBormann"]
+all = ["JohansenCointegration"]
+
+from sktime.param_est.base import BaseParamFitter
+
+
+class JohansenCointegration(BaseParamFitter):
+    """Test for cointegration ranks/relationships for VECM Time-Series.
+
+    Direct interface to ``statsmodels.tsa.vector_ar.vecm``.
+    Determines the coint parameter value to be used in VECM time-series module vecm.py:
+    `coint_rank`. Uses trace statistics or eigenvalue.
+
+    Parameters
+    ----------
+    det_order : int, default=1
+        * -1 - no deterministic terms
```

Comment:
please ensure there are empty lines (two newlines) before and after bullet point lists